python_apps/api_clients/api_clients/api_client.py

```diff
@@ -26,19 +26,19 @@
 # instead of copy pasting them around
 
 def to_unicode(obj, encoding='utf-8'):
-    if isinstance(obj, basestring):
-        if not isinstance(obj, unicode):
-            obj = unicode(obj, encoding)
+    if isinstance(obj, str):
+        if not isinstance(obj, str):
+            obj = str(obj, encoding)
     return obj
 
 def encode_to(obj, encoding='utf-8'):
-    if isinstance(obj, unicode):
+    if isinstance(obj, str):
         obj = obj.encode(encoding)
     return obj
```

Comment:
I also think that `encode_to` is not needed, as we don't want to work with `bytes`, but `string`.